packages/dev/bundle-stats-cli/package.json

```
@@ -1,7 +1,18 @@
 {
   "name": "@parcel/bundle-stats-cli",
```

```suggestion
  "name": "@parcel/bundle-stats",
```

This would have been the only CLI with "cli" in it's name 🤷